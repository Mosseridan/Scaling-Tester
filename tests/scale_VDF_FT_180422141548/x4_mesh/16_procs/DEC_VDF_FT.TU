Statistiques de post resolution

Temps total                       1.87658

