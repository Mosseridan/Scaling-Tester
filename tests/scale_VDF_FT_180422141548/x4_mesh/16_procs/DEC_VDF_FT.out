-------------------------------------------------------------------
 
                          TRUST
                      version : 1.7.6 
                          CEA - DEN
 

 0 file(s) modified 


 405 file(s) created 

 ------------------------------------------------------------------
 
  Vous traitez le cas DEC_VDF_FT
 
Reading data file 

14 MBytes of RAM taken by the calculation.
15 MBytes of RAM taken by the calculation.
16 MBytes of RAM taken by the calculation.
16 MBytes of RAM taken by the calculation.
 Runtime parameters:
   Objective type: METIS_OBJTYPE_CUT
   Coarsening type: METIS_CTYPE_SHEM
   Initial partitioning type: METIS_IPTYPE_GROW
   Refinement type: METIS_RTYPE_FM
   Perform a 2-hop matching: No
   Number of balancing constraints: 1
   Number of refinement iterations: 10
   Random number seed: -1
   Number of partitions: 16
   Number of cuts: 1
   User-supplied ufactor: 1
   Target partition weights: 
        0=[6.25e-02]   1=[6.25e-02]   2=[6.25e-02]   3=[6.25e-02]   4=[6.25e-02]
        5=[6.25e-02]   6=[6.25e-02]   7=[6.25e-02]   8=[6.25e-02]   9=[6.25e-02]
       10=[6.25e-02]  11=[6.25e-02]  12=[6.25e-02]  13=[6.25e-02]  14=[6.25e-02]
       15=[6.25e-02]
   Allowed maximum load imbalance: 1.001 

     16000      92000      92000 [20] [     1200:   16000 ]
      8371      67350      76742 [20] [     1200:   16000 ]
      4383      43828      64964 [20] [     1200:   16000 ]
      2296      26120      55448 [20] [     1200:   16000 ]
      1194      14656      46956 [20] [     1200:   16000 ]
       622       7840      39796 [20] [     1200:   16000 ]
       322       4070      33642 [20] [     1200:   16000 ]
       165       2004      28164 [20] [     1200:   16000 ]
        84        964      23324 [20] [     1200:   16000 ]
        44        460      19158 [20] [     1200:   16000 ]
        23        194      15168 [20] [     1200:   16000 ]
        20        158      14698 [20] [     1200:   16000 ]
      8000      45600      45600 [20] [      600:    8000 ]
      4184      33326      37968 [20] [      600:    8000 ]
      2197      21752      32068 [20] [      600:    8000 ]
      1150      12934      27282 [20] [      600:    8000 ]
       600       7210      23056 [20] [      600:    8000 ]
       311       3824      19526 [20] [      600:    8000 ]
       162       1958      16338 [20] [      600:    8000 ]
        85        946      13664 [20] [      600:    8000 ]
        45        444      11294 [20] [      600:    8000 ]
        23        196       9196 [20] [      600:    8000 ]
        18        146       8818 [20] [      600:    8000 ]
      4000      22400      22400 [20] [      300:    4000 ]
      2093      16284      18586 [20] [      300:    4000 ]
      1107      10502      15690 [20] [      300:    4000 ]
       579       6176      13280 [20] [      300:    4000 ]
       304       3402      11216 [20] [      300:    4000 ]
       160       1820       9480 [20] [      300:    4000 ]
        82        894       7844 [20] [      300:    4000 ]
        44        428       6404 [20] [      300:    4000 ]
        23        194       5200 [20] [      300:    4000 ]
        19        154       4984 [20] [      300:    4000 ]
      2000      11000      11000 [20] [      150:    2000 ]
      1050       7984       9100 [20] [      150:    2000 ]
       554       5180       7658 [20] [      150:    2000 ]
       290       2982       6472 [20] [      150:    2000 ]
       152       1622       5380 [20] [      150:    2000 ]
        80        832       4492 [20] [      150:    2000 ]
        42        408       3788 [20] [      150:    2000 ]
        21        176       2998 [20] [      150:    2000 ]
        21        176       2998 [20] [      150:    2000 ]
      2000      11000      11000 [20] [      150:    2000 ]
      1049       7944       9098 [20] [      150:    2000 ]
       551       5124       7664 [20] [      150:    2000 ]
       286       2966       6518 [20] [      150:    2000 ]
       148       1612       5414 [20] [      150:    2000 ]
        78        832       4540 [20] [      150:    2000 ]
        39        404       3716 [20] [      150:    2000 ]
        21        178       3082 [20] [      150:    2000 ]
        19        162       3012 [20] [      150:    2000 ]
      4000      22400      22400 [20] [      300:    4000 ]
      2091      16240      18582 [20] [      300:    4000 ]
      1091      10410      15618 [20] [      300:    4000 ]
       566       6100      13214 [20] [      300:    4000 ]
       294       3308      11044 [20] [      300:    4000 ]
       153       1728       9272 [20] [      300:    4000 ]
        81        884       7720 [20] [      300:    4000 ]
        43        418       6380 [20] [      300:    4000 ]
        24        200       5126 [20] [      300:    4000 ]
        18        146       4814 [20] [      300:    4000 ]
      2000      11000      11000 [20] [      150:    2000 ]
      1039       7906       9078 [20] [      150:    2000 ]
       545       5040       7568 [20] [      150:    2000 ]
       285       2934       6402 [20] [      150:    2000 ]
       149       1574       5284 [20] [      150:    2000 ]
        78        810       4370 [20] [      150:    2000 ]
        41        390       3698 [20] [      150:    2000 ]
        21        168       2964 [20] [      150:    2000 ]
        19        146       2906 [20] [      150:    2000 ]
      2000      11000      11000 [20] [      150:    2000 ]
      1043       7902       9086 [20] [      150:    2000 ]
       544       5022       7598 [20] [      150:    2000 ]
       284       2906       6400 [20] [      150:    2000 ]
       147       1548       5358 [20] [      150:    2000 ]
        78        794       4506 [20] [      150:    2000 ]
        40        374       3656 [20] [      150:    2000 ]
        21        174       2908 [20] [      150:    2000 ]
        19        158       2848 [20] [      150:    2000 ]
      8000      45600      45600 [20] [      600:    8000 ]
      4176      33394      37952 [20] [      600:    8000 ]
      2192      21720      32146 [20] [      600:    8000 ]
      1142      12914      27254 [20] [      600:    8000 ]
       593       7156      23072 [20] [      600:    8000 ]
       306       3774      19476 [20] [      600:    8000 ]
       158       1898      16336 [20] [      600:    8000 ]
        82        944      13606 [20] [      600:    8000 ]
        42        444      11340 [20] [      600:    8000 ]
        22        184       9096 [20] [      600:    8000 ]
        19        164       8880 [20] [      600:    8000 ]
      4000      22400      22400 [20] [      300:    4000 ]
      2090      16380      18580 [20] [      300:    4000 ]
      1094      10532      15732 [20] [      300:    4000 ]
       575       6216      13354 [20] [      300:    4000 ]
       300       3398      11260 [20] [      300:    4000 ]
       155       1764       9410 [20] [      300:    4000 ]
        80        884       7848 [20] [      300:    4000 ]
        42        444       6498 [20] [      300:    4000 ]
        23        194       5366 [20] [      300:    4000 ]
        19        158       5072 [20] [      300:    4000 ]
      2000      10980      10980 [20] [      150:    2000 ]
      1043       7914       9066 [20] [      150:    2000 ]
       551       5124       7624 [20] [      150:    2000 ]
       289       2966       6448 [20] [      150:    2000 ]
       151       1606       5402 [20] [      150:    2000 ]
        81        858       4514 [20] [      150:    2000 ]
        43        416       3704 [20] [      150:    2000 ]
        23        198       2984 [20] [      150:    2000 ]
        19        156       2834 [20] [      150:    2000 ]
      2000      10960      10960 [20] [      150:    2000 ]
      1047       7958       9054 [20] [      150:    2000 ]
       551       5058       7596 [20] [      150:    2000 ]
       292       2992       6406 [20] [      150:    2000 ]
       153       1568       5388 [20] [      150:    2000 ]
        81        812       4516 [20] [      150:    2000 ]
        42        380       3644 [20] [      150:    2000 ]
        23        176       2902 [20] [      150:    2000 ]
        19        146       2808 [20] [      150:    2000 ]
      4000      22400      22400 [20] [      300:    4000 ]
      2092      16390      18584 [20] [      300:    4000 ]
      1107      10678      15704 [20] [      300:    4000 ]
       581       6294      13308 [20] [      300:    4000 ]
       299       3424      11184 [20] [      300:    4000 ]
       154       1758       9320 [20] [      300:    4000 ]
        82        884       7734 [20] [      300:    4000 ]
        43        424       6300 [20] [      300:    4000 ]
        23        196       5204 [20] [      300:    4000 ]
        20        166       5128 [20] [      300:    4000 ]
      2000      11000      11000 [20] [      150:    2000 ]
      1049       7986       9098 [20] [      150:    2000 ]
       554       5106       7632 [20] [      150:    2000 ]
       289       2962       6454 [20] [      150:    2000 ]
       151       1574       5380 [20] [      150:    2000 ]
        78        804       4482 [20] [      150:    2000 ]
        40        382       3662 [20] [      150:    2000 ]
        21        170       2916 [20] [      150:    2000 ]
        18        138       2808 [20] [      150:    2000 ]
      2000      11000      11000 [20] [      150:    2000 ]
      1046       7972       9092 [20] [      150:    2000 ]
       551       5142       7638 [20] [      150:    2000 ]
       285       2940       6434 [20] [      150:    2000 ]
       148       1578       5372 [20] [      150:    2000 ]
        77        792       4446 [20] [      150:    2000 ]
        40        376       3538 [20] [      150:    2000 ]
        22        172       2828 [20] [      150:    2000 ]
        18        136       2768 [20] [      150:    2000 ]

Timing Information -------------------------------------------------
 Multilevel: 		   0.061
     Coarsening: 		   0.027
            Matching: 			   0.005
            Contract: 			   0.012
     Initial Partition: 	   0.000
     Uncoarsening: 		   0.032
          Refinement: 			   0.005
          Projection: 			   0.028
     Splitting: 		   0.001
********************************************************************

 gk_mcore statistics
           coresize:       193032         nmops:         2048  cmop:      0
        num_callocs:         1481   num_hallocs:            0
       size_callocs:      3690944  size_hallocs:            0
        cur_callocs:            0   cur_hallocs:            0
        max_callocs:       192016   max_hallocs:            0
 nbrpool statistics
        nbrpoolsize:            0   nbrpoolcpos:            0
    nbrpoolreallocs:            0

 Runtime parameters:
   Objective type: METIS_OBJTYPE_CUT
   Coarsening type: METIS_CTYPE_SHEM
   Initial partitioning type: METIS_IPTYPE_GROW
   Refinement type: METIS_RTYPE_FM
   Perform a 2-hop matching: No
   Number of balancing constraints: 1
   Number of refinement iterations: 10
   Random number seed: -1
   Number of partitions: 16
   Number of cuts: 1
   User-supplied ufactor: 1
   Target partition weights: 
        0=[6.25e-02]   1=[6.25e-02]   2=[6.25e-02]   3=[6.25e-02]   4=[6.25e-02]
        5=[6.25e-02]   6=[6.25e-02]   7=[6.25e-02]   8=[6.25e-02]   9=[6.25e-02]
       10=[6.25e-02]  11=[6.25e-02]  12=[6.25e-02]  13=[6.25e-02]  14=[6.25e-02]
       15=[6.25e-02]
   Allowed maximum load imbalance: 1.001 

       800       3840       3840 [20] [       60:     800 ]
       420       2660       3080 [20] [       60:     800 ]
       220       1594       2488 [20] [       60:     800 ]
       116        830       2020 [20] [       60:     800 ]
        63        396       1612 [20] [       60:     800 ]
        33        174       1166 [20] [       60:     800 ]
        19         84        888 [20] [       60:     800 ]
       400       1876       1876 [20] [       30:     400 ]
       212       1294       1500 [20] [       30:     400 ]
       113        784       1212 [20] [       30:     400 ]
        61        410        968 [20] [       30:     400 ]
        32        190        728 [20] [       30:     400 ]
        18         84        554 [20] [       30:     400 ]
       200        916        916 [20] [       15:     200 ]
       105        622        726 [20] [       15:     200 ]
        55        344        580 [20] [       15:     200 ]
        29        172        448 [20] [       15:     200 ]
        19        104        382 [20] [       15:     200 ]
       100        432        432 [20] [        7:     100 ]
        52        290        336 [20] [        7:     100 ]
        29        162        270 [20] [        7:     100 ]
        20        106        232 [20] [        7:     100 ]
       100        432        432 [20] [        7:     100 ]
        52        288        336 [20] [        7:     100 ]
        28        154        264 [20] [        7:     100 ]
        21        116        246 [20] [        7:     100 ]
        21        116        246 [20] [        7:     100 ]
       200        912        912 [20] [       15:     200 ]
       106        642        724 [20] [       15:     200 ]
        57        380        586 [20] [       15:     200 ]
        31        186        462 [20] [       15:     200 ]
        18         92        370 [20] [       15:     200 ]
       100        432        432 [20] [        7:     100 ]
        52        292        336 [20] [        7:     100 ]
        28        162        274 [20] [        7:     100 ]
        22        124        254 [20] [        7:     100 ]
        22        124        254 [20] [        7:     100 ]
       100        440        440 [20] [        7:     100 ]
        54        300        348 [20] [        7:     100 ]
        29        182        276 [20] [        7:     100 ]
        19        110        234 [20] [        7:     100 ]
       400       1872       1872 [20] [       30:     400 ]
       207       1250       1486 [20] [       30:     400 ]
       109        740       1176 [20] [       30:     400 ]
        59        380        940 [20] [       30:     400 ]
        31        170        706 [20] [       30:     400 ]
        18         86        564 [20] [       30:     400 ]
       200        916        916 [20] [       15:     200 ]
       106        622        728 [20] [       15:     200 ]
        54        352        582 [20] [       15:     200 ]
        29        180        466 [20] [       15:     200 ]
        15         74        366 [20] [       15:     200 ]
       100        436        436 [20] [        7:     100 ]
        54        302        344 [20] [        7:     100 ]
        30        170        280 [20] [        7:     100 ]
        18         98        236 [20] [        7:     100 ]
       100        440        440 [20] [        7:     100 ]
        51        290        342 [20] [        7:     100 ]
        27        156        262 [20] [        7:     100 ]
        22        128        244 [20] [        7:     100 ]
        22        128        244 [20] [        7:     100 ]
       200        916        916 [20] [       15:     200 ]
       105        610        726 [20] [       15:     200 ]
        56        380        584 [20] [       15:     200 ]
        31        196        476 [20] [       15:     200 ]
        17         92        358 [20] [       15:     200 ]
       100        436        436 [20] [        7:     100 ]
        52        288        340 [20] [        7:     100 ]
        28        158        266 [20] [        7:     100 ]
        20        110        242 [20] [        7:     100 ]
       100        428        428 [20] [        7:     100 ]
        52        274        332 [20] [        7:     100 ]
        28        158        252 [20] [        7:     100 ]
        21        114        226 [20] [        7:     100 ]
        21        114        226 [20] [        7:     100 ]

Timing Information -------------------------------------------------
 Multilevel: 		   0.010
     Coarsening: 		   0.002
            Matching: 			   0.001
            Contract: 			   0.001
     Initial Partition: 	   0.003
     Uncoarsening: 		   0.004
          Refinement: 			   0.003
          Projection: 			   0.001
     Splitting: 		   0.001
********************************************************************

 gk_mcore statistics
           coresize:        10632         nmops:         2048  cmop:      0
        num_callocs:          870   num_hallocs:            0
       size_callocs:       208272  size_hallocs:            0
        cur_callocs:            0   cur_hallocs:            0
        max_callocs:         9616   max_hallocs:            0
 nbrpool statistics
        nbrpoolsize:            0   nbrpoolcpos:            0
    nbrpoolreallocs:            0

18 MBytes of RAM taken by the calculation.

--------------------------------------------
clock: Total execution: 1.87658 s
